seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-replace/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>seatunnel-transforms-flink</artifactId>
+        <groupId>org.apache.seatunnel</groupId>
+        <version>2.1.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>seatunnel-transform-flink-replace</artifactId>
+
+    <properties>
+        <junit.version>4.4</junit.version>
```

Comment:
We have defined the JUnit version in root `pom.xml`, so you don't need to define a version here, this will make it hard to manage the version.

Reply:
ok, I have modified